app/models/maintenance_stats/queries/github/commit_count_query.rb

```diff
@@ -29,7 +38,18 @@ def self.client_type
         def query(params: {})
           validate_params(params)
 
-          @client.query(COMMIT_COUNTS_QUERY, variables: params)
+          # generate the four dates needed
+          date_params = Hash.new { |k,v| k[v] = v.iso8601 }
```

Comment:
The arguments to the block are the hash and the new key - I think you're going to end up with `:one_week.iso8601` here?

Reply:
Yeah I don't think this actually works the way I thought it would. I'm going to remove since it isn't even being used.